bitnami/metallb/Chart.yaml

```diff
@@ -30,4 +30,4 @@ sources:
   - https://github.com/metallb/metallb
   - https://github.com/bitnami/bitnami-docker-metallb
   - https://metallb.universe.tf
-version: 2.3.7
+version: 2.3.8
```

Comment:
Could you increase this as a minor version (i.e. `2.3.7 -> 2.4.0`)? Since we are adding new features